src/ui/flutter_app/lib/shared/constants.dart

```diff
@@ -24,7 +24,7 @@ class Constants {
   static const String authorAccount = "calcitem";
   static const String projectName = "Sanmill";
   static String projectNameLower = projectName.toLowerCase();
-  static const String recipients = "$authorAccount@outlook.com";
+  static const List<String> recipients = ["$authorAccount@outlook.com"];
```

Comment:
You will receive some cursing emails. It is better for me to set up automatic forwarding or manual forwarding.

Reply:
I did not change the logic of the code. 
The variable name is `recipients` (plural) so the content of the variable should represent that (List<String>)